docker/luna_tutorial/.dockerignore

```diff
@@ -0,0 +1,7 @@
+*/*.svs
+*/*.tar
+*/*.parquet
+*/*.pt
+*/*.pil
+*/*.npy
+*/*.bmp
```

Comment:
Looks like `.ipynb_checkpoints` folders are checked in. Should this be added in .dockerignore

Reply:
Removed checkout dir from revision control.